frontend/src/features/homes/home-group.vue

```diff
@@ -0,0 +1,190 @@
+        <template v-slot:body-cell-activityLevel="props">
+          <q-td :props="props">
+            <div :id="`activityLevelCountsChart-${props.row._id}`"></div>
+          </q-td>
+        </template>
+
+        <template v-slot:header-cell-name="props">
+          <q-th :props="props">
+            <q-icon name="fa fa-home" class="q-mr-sm" />{{ props.col.label }}
+          </q-th>
+        </template>
+
+        <template v-slot:header-cell-activityLevel="props">
+          <q-th :props="props">
+            {{ props.col.label }}
+            <div class="float-right">
+              <q-icon class="legend-inactive" name="fa fa-square" />&nbsp;
+              {{ $i18n.t("homeGroup-activityLevelsLegend-inactive") }}
+              <q-icon class="legend-semiactive" name="fa fa-square" />&nbsp;
+              {{ $i18n.t("homeGroup-activityLevelsLegend-semiActive") }}
+              <q-icon class="legend-active" name="fa fa-square" />&nbsp;
+              {{ $i18n.t("homeGroup-activityLevelsLegend-active") }}
+            </div>
+          </q-th>
+        </template>
+
+        <template v-slot:body-cell-viewHome="props">
+          <q-td :props="props">
+            <q-btn size="sm" outline class="text-black bg-white">
+              <q-icon name="fa fa-eye" class="q-mr-sm" />
+              {{ $i18n.t("homeGroup-home-viewButton-text") }}
+            </q-btn>
+          </q-td>
+        </template>
+
+        <template v-slot:body-cell-viewReport="props">
+          <q-td :props="props">
+            <q-btn size="sm" outline class="text-black bg-white">
+              <q-icon name="fa fa-chart-line" class="q-mr-sm" />
+              {{ $i18n.t("homeGroup-home-reportButton-text") }}
+            </q-btn>
+          </q-td>
+        </template>
+      </q-table>
+    </q-card-section>
+
+    <add-home v-if="showAddHome" :group-id="group._id" @close="close" />
+    <assign-manager
+      v-if="showAssignManager"
+      :group-id="group._id"
+      @close="closeManager"
+    />
+  </q-card>
+</template>
+
+<script>
+import { getHomesWithActivityLevel } from "./services/homes-list";
+import { renderChart } from "./services/home-activity-levels.chart";
+import AddHome from "./add-home";
+import AssignManager from "./assign-manager";
+
+export default {
+  props: {
+    group: { type: Object, required: true }
+  },
+
+  components: {
+    AddHome,
+    AssignManager
+  },
+
+  data() {
+    return {
+      homes: [],
+      loading: false,
+      showAddHome: false,
+      showAssignManager: false,
+      columns: [
+        {
+          name: "viewHome",
+          align: "left",
+          field: "_id",
+          label: this.$i18n.t("homeGroup-tableHeader-viewHome")
+        },
+        {
+          name: "name",
+          align: "left",
+          field: "name",
+          label: this.$i18n.t("homeGroup-tableHeader-name")
+        },
+        {
+          name: "activityLevel",
+          align: "left",
+          style: "width: 600px",
```

Comment:
Can this be expressed in relative units, such as `em` or `rem` values?

Reply:
done